frontend/src/components/dashboard/UserDetails.tsx

```
@@ -0,0 +1,23 @@
+import { Card, CardContent, CardHeader } from "../ui/card"
+import { Input } from "../ui/input"
+
+const UserDetails = () => {
+  return (
+    <div>
+      <Card className="p-6 max-w-3xl">
+          <CardHeader className="mb-4">
+            <label className="font-medium">Username</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)